actors/evm/src/interpreter/instructions/lifecycle.rs

```diff
@@ -127,6 +123,10 @@ fn create_init<'r, BS: Blockstore, RT: Runtime<BS>>(
     // send bytecode & params to EAM to generate the address and contract
     let ret = platform.rt.send(&EAM_ACTOR_ADDR, method, params, value);
 
+    // Overall future work / TODOs:
+    //  readonly state things | https://github.com/filecoin-project/ref-fvm/issues/971
+    //  preload state items (~eq to EVM access list) | maybe M3
```

Comment:
Let's just remove these.

- Preloading is an FVM feature.
- Readonly state things will touch a _lot_ of EVM parts (but likely won't actually be relevent to the EAM).

TODOs should generally be actionable and directly relevant, otherwise they're confusing. In this case, I'm never going to look at these TODOs and decide to do something about them, because they're not relevant to this part of the code.